configuration/concepts/program_attribute_types.csv

```diff
@@ -23,10 +23,16 @@ bef52504-d77b-485c-ab81-9ccb6eb8f239,,,Pediatrics,Pediatrics,,,,,Misc,N/A,,,,,,,
 dc04cfe1-29d1-46a7-a0cf-b8baacfdbb31,,,Dermatology,Dermatology,,,,,Misc,N/A,,,,,,,,,,,
 13aa8824-8a29-46c9-b80e-9201f03e82e5,,,Wound,Wound,,,,,Misc,N/A,,,,,,,,,,,
 a20c6c3b-d504-43e8-9ced-bbf788a1f9a8,,,Goldman Score,Goldman Score,,,,,Misc,Coded,,One;Two;Three;Four,,,,,,,,,
+5c5d8fe4-27d8-4a40-832d-506cdc337954,,,Ongoing Care,Ongoing Care,,,,,Misc,N/A,,,,,,,,,,,
+651c5bd1-debd-4485-9bc8-33dd8394775a,,,Surgical Optimization,Surgical Optimization,,,,,Misc,N/A,,,,,,,,,,,
+2873ed4a-c68c-43a8-aede-7e8f6f6dd602,,,Regular Followup,Regular Followup,,,,,Misc,N/A,,,,,,,,,,,
 b392a997-15b7-44b4-819b-8f8db23047aa,,,Consults Service,Consults Service,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental,,,,,,,,,
 5f16a893-d300-4634-9c45-9485be023f98,,,Primary Care Service,Primary Care Service,,,,,Misc,Coded,,Family Medicine;Internal Medicine;Pediatrics;Cardiology;Dermatology;Gynecology;Dental;Ophthalmology;Wound;Other,,,,,,,,,
 d47ce963-77fa-494a-ab37-7f0806d41ecb,,,Priority 1,Priority 1,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental,,,,,,,,,
 c69ff88e-309b-41a0-99ae-5eb58dfc5966,,,Priority 2,Priority 2,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental,,,,,,,,,
 8a6d5b95-ac55-41c2-b27e-ce18012bf962,,,Priority 3,Priority 3,,,,,Misc,Coded,,Upper extremity;Lower extremity;Orthopedic/Peds;Sports Orthopedics;Foot/ankle;Podiatry;Total Hip;Total Knee;Shoulder;Spine;Surgical Oncology;General Surgery;ENT;Urology;Gynecology;Ophthalmology;Dental,,,,,,,,,
 3f8022b3-5d0e-4b1f-ac4e-89b970017edb,,,Done,Done,,,,,Misc,N/A,,,,,,,,,,,
-63600df3-1525-40b1-8336-e312f649c68f,,,Program outcome,Program outcome,,,,,ConvSet,N/A,,,Done,,,,,,,,
\ No newline at end of file
+63600df3-1525-40b1-8336-e312f649c68f,,,Program outcome,Program outcome,,,,,ConvSet,N/A,,,Done,,,,,,,,
+8546be29-e249-4d1a-8f5c-abe0fb941d4a,,,Status,Status,,,,,Misc,Coded,,Ongoing Care;Surgical Optimization;Regular Followup,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,
```

Comment:
Remove trailing commas

Reply:
Let us rename it to 'Primary Care Status' but keep the short name as 'Status' to avoid conflict with this [PR](https://github.com/globalhealthcoalition/openmrs-config-ghc/pull/61)